Cannot create export profiles - SQL Error

### What happened?

I have done an upgrade from Freeform 4 to Freeform 5 and I am trying to setup some new export profiles. However, I am getting the following error;

```
Integrity constraint violation - yiildblIntegrityException
SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'rangeStart' cannot be null
The SQL being executed was: INSERT INTO 'craft_freeform_export_profiles (name', "formld", 'limit", 'dateRange, 'rangeStart', 'rangeEnd', 'fields', 'filters', 'statuses', dateCreated*,
"dateUpdated", 'uid') VALUES ('Holiday Courses', 16, NULL, NULL, NULL, NULL, '"idl": ("checked": I"1\", 'labell": "IDI", "titlel":("checked": ("1", l"labell" "Titlel"}, "ipl":
kd""", l"labell"" Address", "datreted"("hecked""" "labell""Date rated", tatusl""hked""" "labell""Statusl" luserld)"،
[1"checkedا":|"1l",|"labell":|"Authorl"}, |"1421":[l"checkedl":|"1l",l"labell":|"Coursesl"), |"143l":[I"checkedl":|"1l", I"labell":|"Total Amountl"!, |"1441":("checkedl":|"1l",l"labell":|"First Namel"?,|"145l":
{1"checkedl":|"11",|"labell":|"Surnamel"}, |"1461":[l"checkedl":|"11", l"labell":|"Email Addressl"}, |"1471":("checkedl":|"11",l"labell":I"Phone Numberl"!, |"148l":["checkedl":|"11",l"labell":I"Childs First
Name'"}, l"1491":("checked":|"1\", l"labell": I"Childs Surnamel",l"1501":("checkedl":|"1\", l"labell":I"I agree to the Terms and Conditions'"}, "1521": {l"checked": "11", l"labell": "Stripel"}}', 'O', **, '2025-12-03 09:46:48', '2025-12-03 09:46:48', '011d57ad-e609-41db-a4b3-8ba653764f7d')
Error Info: Array
1044
'rangeStart' cannot be null
→ Caused by: PDOException
SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'rangeStart' cannot be null in /home/ploi/dev.hatchendtuition.co.uk/vendor/yiisoft/yii2/db/Command.php at line 1320
```

<img width="1949" height="1582" alt="Image" src="https://github.com/user-attachments/assets/dd6010f9-90df-4e6a-a438-530d706d2a06" />


### Errors and Stack Trace (if available)

```shell
Integrity constraint violation - yiildblIntegrityException
SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'rangeStart' cannot be null
The SQL being executed was: INSERT INTO 'craft_freeform_export_profiles (name', "formld", 'limit", 'dateRange, 'rangeStart', 'rangeEnd', 'fields', 'filters', 'statuses', dateCreated*,
"dateUpdated", 'uid') VALUES ('Holiday Courses', 16, NULL, NULL, NULL, NULL, '"idl": ("checked": I"1\", 'labell": "IDI", "titlel":("checked": ("1", l"labell" "Titlel"}, "ipl":
kd""", l"labell"" Address", "datreted"("hecked""" "labell""Date rated", tatusl""hked""" "labell""Statusl" luserld)"،
[1"checkedا":|"1l",|"labell":|"Authorl"}, |"1421":[l"checkedl":|"1l",l"labell":|"Coursesl"), |"143l":[I"checkedl":|"1l", I"labell":|"Total Amountl"!, |"1441":("checkedl":|"1l",l"labell":|"First Namel"?,|"145l":
{1"checkedl":|"11",|"labell":|"Surnamel"}, |"1461":[l"checkedl":|"11", l"labell":|"Email Addressl"}, |"1471":("checkedl":|"11",l"labell":I"Phone Numberl"!, |"148l":["checkedl":|"11",l"labell":I"Childs First
Name'"}, l"1491":("checked":|"1\", l"labell": I"Childs Surnamel",l"1501":("checkedl":|"1\", l"labell":I"I agree to the Terms and Conditions'"}, "1521": {l"checked": "11", l"labell": "Stripel"}}', 'O', **, '2025-12-03 09:46:48', '2025-12-03 09:46:48', '011d57ad-e609-41db-a4b3-8ba653764f7d')
Error Info: Array
1044
'rangeStart' cannot be null
→ Caused by: PDOException
SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'rangeStart' cannot be null in /home/ploi/dev.hatchendtuition.co.uk/vendor/yiisoft/yii2/db/Command.php at line 1320
```

### How can we reproduce this?

1. Create a new export profile for a form
2. Enter a title
3. Click Save


### Freeform Edition

Pro

### Freeform Version

5.13.6

### Craft Version

Craft Pro 4.16.16

### When did this issue start?

- [ ] Unsure
- [ ] Fresh install of Freeform
- [x] After upgrading from older Freeform version
- [ ] After upgrading from older Craft version
- [ ] After a change to hosting configuration

### Previous Freeform Version

4.1.28

--- a/packages/plugin/src/migrations/m251204_120000_AllowNullExportProfileRanges.php
+++ b/packages/plugin/src/migrations/m251204_120000_AllowNullExportProfileRanges.php
@@ -1,0 +1,34 @@
+<?php
+
+namespace Solspace\Freeform\migrations;
+
+use craft\db\Migration;
+
+class m251204_120000_AllowNullExportProfileRanges extends Migration
+{
+    public function safeUp(): bool
+    {
+        $table = '{{%freeform_export_profiles}}';
+        if (!$this->db->tableExists($table)) {
+            return true;
+        }
+
+        $schema = $this->db->getTableSchema($table, true);
+        if ($schema && isset($schema->columns['rangeStart'])) {
+            $this->alterColumn($table, 'rangeStart', $this->string(255)->null());
+        }
+
+        if ($schema && isset($schema->columns['rangeEnd'])) {
+            $this->alterColumn($table, 'rangeEnd', $this->string(255)->null());
+        }
+
+        return true;
+    }
+
+    public function safeDown(): bool
+    {
+        echo "m251204_120000_AllowNullExportProfileRanges cannot be reverted.\n";
+
+        return false;
+    }
+}

